public/src/renderers/post_renderer.ts

```diff
@@ -36,4 +38,16 @@ export class PostRenderer {
 function getImageSrc(post: Post): Promise<string> {
     const imageRef = firebaseApp.storage().ref(post.url);
     return imageRef.getDownloadURL();
+}
+
+// TODO: Adjust for better rendering of the audience (could run into errors if a groupId and userId are shared)
+async function getAudienceName(audience: string) {
+    var currentUser = await new UserService().getCurrentUser();
+    if (audience == "public") {
+        return "Public";
+    } else if (audience == currentUser.id) {
+        return "Private";
+    } else {
+        return "Group";
```

Comment:
instead of "Group", we can just use the audience string so that it shows up as "cats"

Reply:
Fixed.